application/public/map-styles/map-style.json

```
@@ -1,1112 +1,1362 @@
-        "text-size": {"base": 1, "stops": [[0, 9], [6, 11]]},
-        "text-transform": "uppercase",
-        "visibility": "visible"
-      },
-      "paint": {
-        "text-color": {
-          "base": 1,
-          "stops": [[3, "rgb(157,169,177)"], [4, "rgb(153, 153, 153)"]]
-        },
-        "text-halo-color": "rgba(236,236,234,0.7)",
-        "text-halo-width": 1.4
-      }
-    },
-    {
-      "id": "place_country_minor",
-      "type": "symbol",
-      "metadata": {"maplibre:group": "101da9f13b64a08fa4b6ac1168e89e5f"},
-      "source": "openmaptiles",
-      "source-layer": "place",
-      "maxzoom": 8,
-      "filter": [
-        "all",
-        ["==", "$type", "Point"],
-        ["==", "class", "country"],
-        [">=", "rank", 2],
-        ["==", "iso_a2", ""]
-      ],
-      "layout": {
-        "text-field": "{name:latin}",
-        "text-font": ["Metropolis Regular", "Noto Sans Regular"],
-        "text-size": {"base": 1, "stops": [[0, 10], [6, 12]]},
-        "text-transform": "uppercase",
-        "visibility": "visible"
-      },
-      "paint": {
-        "text-color": {
-          "base": 1,
-          "stops": [[3, "rgb(157,169,177)"], [4, "rgb(153, 153, 153)"]]
-        },
-        "text-halo-color": "rgba(236,236,234,0.7)",
-        "text-halo-width": 1.4
-      }
-    },
-    {
-      "id": "place_country_major",
-      "type": "symbol",
-      "metadata": {"maplibre:group": "101da9f13b64a08fa4b6ac1168e89e5f"},
-      "source": "openmaptiles",
-      "source-layer": "place",
-      "maxzoom": 6,
-      "filter": [
-        "all",
-        ["==", "$type", "Point"],
-        ["<=", "rank", 1],
-        ["==", "class", "country"],
-        ["==", "iso_a2", ""]
-      ],
-      "layout": {
-        "text-anchor": "center",
-        "text-field": "{name:latin}",
-        "text-font": ["Metropolis Regular", "Noto Sans Regular"],
-        "text-size": {"base": 1.4, "stops": [[0, 10], [3, 12], [4, 14]]},
-        "text-transform": "uppercase",
-        "visibility": "visible"
-      },
-      "paint": {
-        "text-color": {
-          "base": 1,
-          "stops": [[3, "rgb(157,169,177)"], [4, "rgb(153, 153, 153)"]]
-        },
-        "text-halo-color": "rgba(236,236,234,0.7)",
-        "text-halo-width": 1.4
-      }
-    }
-  ],
-  "id": "positron"
+	"version": 8,
+	"name": "Fiord Color",
+	"metadata": {
+		"maplibre:autocomposite": false,
+		"maplibre:groups": {
+			"101da9f13b64a08fa4b6ac1168e89e5f": {
+				"collapsed": false,
+				"name": "Places"
+			},
+			"a14c9607bc7954ba1df7205bf660433f": { "name": "Boundaries" },
+			"b6371a3f2f5a9932464fa3867530a2e5": {
+				"collapsed": false,
+				"name": "Transportation"
+			}
+		},
+		"maplibre:type": "template",
+		"openmaptiles:maplibre:owner": "openmaptiles",
+		"openmaptiles:maplibre:source:url": "maplibre://openmaptiles.4qljc88t",
+		"openmaptiles:version": "3.x",
+		"maputnik:renderer": "mlgljs"
```

D'après le fichier original de datagouv, je pense que tu peux enlever les lignes suivantes :
```suggestion
		"openmaptiles:version": "3.x",
```

---

Tu as supprimé `"openmaptiles:version": "3.x"` en trop  ^^